pkg/test/framework/components/istio/operator.go

```diff
@@ -256,6 +268,10 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 
 	// Wait for all of the control planes to be started.
 	for _, cluster := range env.KubeClusters {
+		// TODO in centralIstiod case, webhook is only installed in control plane cluster
+		if !env.IsControlPlaneCluster(cluster) && isCentralIstio(env, cfg) {
```

Comment:
Why are we waiting for remote clusters as well? Maybe we should just be waiting on clusters with actual control planes in the first place?

cc @stevenctl WDYT?

Reply:
in general multicluster / shared control plane scenario, Istiod is still installed to the remote cluster.  In central Istiod case, only ingress gateway is installed.  Webhook is a good check, but maybe just checking if all pods in Istio-system namespace are running is more appropriate?